src/features/features.tsx

```diff
@@ -84,6 +86,13 @@ const FEATURES: Features = {
     ui: <AspectRatioUi parents={['ASPECT_RATIO']} />
   },
 
+  CHANGE_VOLUME: {
+    name: 'Change-Volume',
+    description: 'Change the volume for the video',
+    feature: ChangeVolumeFeature,
+    ui: <ChangeVolumeUi parents={['CHANGE_VOLUME']} />
```

Comment:
This parent name is wrong it should be the same as in your config, so it should be `VOLUME`